### ISSUE TITLE
Coveralls failing causes the tests to fail

### ISSUE THREAD

**Issue by tofumatt** [mozilla/addons-frontend#3012]
https://travis-ci.org/mozilla/addons-frontend/jobs/268732728 failed because coveralls ran into a server error. We should possibly run coveralls in a separate task, if possible, so we can tell it's failing more obviously.

**Comment by willdurand** [comment]
Confirmed here too: https://travis-ci.org/mozilla/addons-frontend/jobs/269122969.

**Comment by kumar303** [comment]
We've had so many problems and the coveralls report is really hard to parse. I say we just try https://codecov.io/
